uniprot-search/src/main/java/org/uniprot/store/search/domain2/SearchFieldsLoader.java

```diff
@@ -21,9 +21,9 @@
 public class SearchFieldsLoader implements SearchItems, SearchFields {
     private List<SearchItem> searchItems = new ArrayList<>();
     private Set<SearchField> searchFields = new HashSet<>();
-    private Set<String> sortFieldNames;
+    private Set<SearchField> sortFields;
```

Comment:
Hi, is this required?, I think it can be easily extracted from list of search fields